beta/src/components/MDX/Sandpack/NavigationBar.tsx

```diff
@@ -39,6 +42,43 @@ const getFileName = (filePath: string): string => {
   return filePath.slice(lastIndexOfSlash + 1);
 };
 
+function SnippetTargetLanguageButton(props: any) {
+  const {children, snippetTargetLanguage} = props;
+  const {
+    setSnippetTargetLanguage,
+    snippetTargetLanguage: currentSnippetTargetLanguage,
+  } = useContext(SnippetTargetLanguageContext);
+
+  const isCurrent = currentSnippetTargetLanguage === snippetTargetLanguage;
+
+  return (
+    <button
+      aria-pressed={isCurrent ? true : undefined}
+      className={classNames(
+        'text-sm text-primary dark:text-primary-dark inline-flex mx-1 border-black',
+        isCurrent && 'border-b-4'
```

Comment:
Tried `aria-pressed:border-b-4` but this didn't work. Either I misunderstood that API or it would require a Tailwind bump which seems excessive.

Reply:
Yea this is available with `3.2.0` https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.2.0 may be we can bump the package? its a minor bump only?